projects/demo/src/modules/components/input-date-range/examples/6/index.html

```diff
@@ -0,0 +1,22 @@
+<form
+    tuiForm="m"
+    [formGroup]="form"
+>
+    <tui-textfield tuiDropdownMobile="Pick a range">
```

Comment:
Should we horizontally align content to center ?
<img height="300" src="https://github.com/user-attachments/assets/fe5ac5e5-1a38-4483-98d1-3f66a6edc381" />